src/components/executive-education-2u/components/CourseSummaryCard.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Card, Image, Row, Col,
+} from '@edx/paragon';
+import { numberWithPrecision } from '../../course/data/utils';
+
+const CourseSummaryCard = ({ courseMetadata, enrollmentCompleted }) => (
+  <Card
+    className="mb-4 course-summary"
+    orientation="horizontal"
+  >
+    <Card.Body>
+      <Card.Header
+        title={(
+          <Image
+            src={courseMetadata.organizationImage}
+          />
+        )}
+      />
+      <Card.Section>
+        <Row>
+          <Col xs={12} lg={{ span: 8, offset: 0 }} className="vertical">
+            <Row>
+              <Col xs={12} lg={{ span: 8, offset: 0 }}>
+                <p className="small font-weight-light text-gray-500">{ courseMetadata.organizationName }</p>
+                <p>{ courseMetadata.title }</p>
+              </Col>
+            </Row>
+          </Col>
+          <Col xs={12} lg={{ span: 4, offset: 0 }}>
+            <div className="course-details">
+              <Row>
+                <Col className="small font-weight-light text-gray-500 justify-content-start">{enrollmentCompleted ? 'Start date:' : 'Available start date:'}</Col>
+                <Col className="justify-content-end"><Row className="justify-content-end mr-2.5">{courseMetadata.startDate}</Row></Col>
+              </Row>
+              <Row>
+                <Col className="small font-weight-light text-gray-500 justify-content-start">Course duration:</Col>
+                <Col className="justify-content-end"><Row className="justify-content-end mr-2.5">{courseMetadata.duration}</Row></Col>
```

Comment:
nit: casing. Mocks show "weeks" versus implementation shows "Week". Should be plural when appropriate.